assets/src/edit-story/panels/size.js

```diff
@@ -45,11 +44,16 @@ function SizePanel( { selectedElements, onSetProperties } ) {
 						height: typeof newWidth === 'number' && lockRatio ? newWidth / ratio : height,
 					} );
 				} }
-				postfix={ _x( 'px', 'pixels, the measurement of size', 'amp' ) }
 				disabled={ isFullbleed }
 			/>
-			<InputGroup
-				label={ __( 'Height', 'amp' ) }
+			<IconSwitch
+				locked={ lockRatio }
+				onClick={ () => {
+					setLockRatio( ! lockRatio );
+				} }
```

Comment:
Move to a `useCallback` and use the functional version of `setState` not relying on this render's value of `lockRatio` but the actual current value.

Reply:
Yes, it makes more sense. Updated